app/docs/0.10.x/auth.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Authentication Reference
+---
+
+# Authentication Reference
+
+Client access to upstream API services is typically controlled by the application and configuration of 
+Kong authentication plugins. Authetication plugins are [documented individually][plugins]. 
+
+## Anonymous Access
+
+Prior to Kong 0.10.x, a given API could be configured to allow **only** authenticated access (by applying an
+[auth plugin][plugins]) or **only** anonymous access - it was not possible to have a given API allow some 
+users to be authenticated and others to access anonymously.
+
+Kong 0.10.x adds the ability to configure a given API to allow **both** authenticated **and** anonymous access. 
+You might use this configuration to grant access to anonymous users with a low rate-limit, and grant access 
```

Comment:
I think you need the dash between "rate" and "limit" to indicate that "low" applies to "rate-limit". Without the dash, it could be mistakenly read as "low rate...limit".